utils/dataconvert/dataconvert.h

```diff
@@ -1271,10 +1273,13 @@ class DataConvert
 
   // convert string to date
   EXPORT static int64_t stringToDate(const std::string& data);
```

Comment:
I suppose it is time to remove all overloads with std::string.

Reply:
I will spent two weeks fixxing build and ironing bugs then. Why do you hate me? ;)

Reply:
Can't be done now.

Reply:
We can delay this for now.